### ISSUE TITLE
Can't initialize S3 from ArcGIS 10.4 machine

### ISSUE THREAD

**Issue by gmorlock** [Esri/OptimizeRasters#33]
I used OptimizeRasters on a local machine running ArcGIS 10.3.1 to convert some DG WV2 files to MRF and copy them to S3.  Then, from an EC2 instance running ArcGIS 10.4, I attempted to clone the MRF files on S3 to my EBS on the EC2 instance.  I get errors that say: Unable to initialize the (S3) upload module! Check module setup/credentials. Quitting..  I've tried other templates to work with S3 and get similar results.  My credential work because I can pick the S3 bucket from the drop downs in the UI.

**Comment by Chamlika** [comment]
Hi,
    Can you tell me if this workflow/similar pattern used to work before? I need to know if this got broken with the recent UI changes that went through to auto popular the cloud bucket names. Will check on this soon and have a follow up email on this. For now if you can, use the command line to avoid this error. Thanks.

-Chamlika

Sent from my Windows Phone
________________________________

**Comment by gmorlock** [comment]
I did some more testing this morning.  On a different Amazon VM with ArcGIS
10.3.1 installed I did the CloneMRF operation using OptimizeRasters version
20160310 and the operation worked.  Next I downloaded and used version
20160428 and it worked.  Next I went back to my VM running ArcGIS 10.4 and
downloaded and used version 20160428 and it failed as before with the
unable to initialize the S3 error.

Greg
